pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "nautobot-circuit-maintenance"
-version = "0.5.0b1"
+version = "0.5.0"
```

Comment:
Let's not automatically bump the release version in every PR to develop. I see there are 5 other open PRs at present - updating the release version should be done when we're ready to cut a new release and only then.